report_barcode_elaphe/__openerp__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+# Odoo, an open source suite of business apps
+# This module copyright (C) 2015 bloopark systems (<http://bloopark.de>).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Report Barcode Elaphe",
+    'summary': """Barcode generation controller using elaphe library.""",
+    'description': """Overwrite the report barcode controller to use a more powerful python library of barcode generation.""",
+    'author': "bloopark systems GmbH & Co. KG",
+    'website': "http://www.bloopark.de",
+    'license': 'AGPL-3',
+    'category': 'Base',
```

Comment:
Maybe @pedrobaeza can help us here. I don't know which category is better. Can you help us, please?